src/openforms/forms/tests/test_api_form_json_schema.py

```diff
@@ -0,0 +1,128 @@
+from django.urls import reverse
+
+from rest_framework.test import APITestCase
+
+from openforms.accounts.tests.factories import UserFactory
+from openforms.formio.constants import DataSrcOptions
+from openforms.forms.tests.factories import (
+    FormDefinitionFactory,
+    FormFactory,
+    FormStepFactory,
+    FormVariableFactory,
+)
+from openforms.variables.constants import FormVariableDataTypes
+
+
+class FormJsonSchemaAPITests(APITestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = UserFactory.create()
+        self.user.is_staff = True
```

Comment:
this belongs in the `setUpTestData` class method - it runs only once for the entire test case so it's more performant as you're doing less database IO in tests

Additionally, we have a `StaffUserFactory` too, but the following is more idiomatic:

```suggestion
        self.user = UserFactory.create(is_staff=True)
```